apps/src/code-studio/pd/professional_learning_landing/professionalLearningCourseProgress.jsx

```diff
@@ -35,13 +18,22 @@ const ProfessionalLearningCourseProgress = React.createClass({
           <h2>
             Online Professional Learning Courses
           </h2>
-          {this.renderProfessionalLearningEnrollments()}
+          <div>
+            {
+              this.props.professionalLearningCourseData.map((plcData, i) => {
+                return (
+                  <PlcEnrollment
+                    key={i}
+                    plcData={plcData}
+                  />
+                );
+              })
+            }
+          </div>
```

Comment:
Suggestion: This might read better if we inline the component and use an expression body for the arrow function:

```jsx
<div>
  {this.props.professionalLearningCourseData.map((plcData, i) => (
    <PlcEnrollment key={i} plcData={plcData}/>
  )}
</div>
```